src/main/java/games/strategy/internal/persistence/serializable/VersionProxy.java

```diff
@@ -0,0 +1,56 @@
+package games.strategy.internal.persistence.serializable;
```

Comment:
Noting package structure, it would be really nice IMO if the proxy could be close to the class that is proxied. What I'm thinking is that it would make it easy to tell which clases have proxies and which do not. It would also be easy to see an example of it and emulate it. This would scatter our proxies, but I think that is okay because the game data is itself a bit scattered to begin with.

What do you think?